tools/travis-setup-db.sh

```diff
@@ -80,19 +76,39 @@ elif [ $DB = 'riak' ]; then
     sudo cp ${SSLDIR}/fake_cert.pem ${RIAK_DIR}/cert.pem
     sudo cp ${SSLDIR}/fake_key.pem ${RIAK_DIR}/key.pem
     sudo cp ${SSLDIR}/ca/cacert.pem ${RIAK_DIR}/cacertfile.pem
-    sudo cp ${RIAK_CONF_DIR}/advanced.config ${RIAK_DIR}/advanced.config
-    sudo cp ${RIAK_CONF_DIR}/riak.conf ${RIAK_DIR}/riak.conf
+    sudo cp ${DB_CONF_DIR}/advanced.config ${RIAK_DIR}/advanced.config
+    sudo cp ${DB_CONF_DIR}/riak.conf ${RIAK_DIR}/riak.conf
     sudo service riak restart
     echo "Setup Riak"
     sudo tools/setup_riak $TRAVIS_DB_PASSWORD
 
 elif [ $DB = 'cassandra' ]; then
-    docker run -d -p 9042:9042 -e MAX_HEAP_SIZE=128M -e HEAP_NEWSIZE=64M --name=cassandra cassandra:${CASSANDRA_VERSION}
+    docker image pull cassandra:${CASSANDRA_VERSION}
+
+    opts="$(docker inspect -f '{{range .Config.Entrypoint}}{{println}}{{.}}{{end}}' cassandra:${CASSANDRA_VERSION})"
+    opts+="$(docker inspect -f '{{range .Config.Cmd}}{{println}}{{.}}{{end}}' cassandra:${CASSANDRA_VERSION})"
+    readarray -t -s 1 init_opts <<< "$opts"
+    echo -e "cassandra startup cmd:\n\t${init_opts[@]}"
+
+    docker_entry="${DB_CONF_DIR}/docker_entry.sh"
+
+    docker run -d -p 9042:9042                   \
+               -e MAX_HEAP_SIZE=128M             \
```

Comment:
Is there a need to set those parameters? If yes, why these particular values? I mean `MAX_HEAP_SIZE` and `HEAP_NEWSIZE`.

Reply:
TBH i have no idea, but these 2 were set before so i left them as is